app/models/slack_methods.rb

```diff
@@ -74,7 +74,13 @@ def self.introduce_incident!(channel_id, tech_lead)
     message = slack_client.chat_postMessage(channel: channel_id,
                                             text: "Welcome to the incident channel. Please review the following docs:\n> <#{ENV['INCIDENT_PLAYBOOK']}|Incident playbook> \n><#{ENV['INCIDENT_CATEGORIES']}|Incident categorisation>")
     slack_client.pins_add(channel: channel_id, timestamp: message[:ts])
-    slack_client.chat_postMessage(channel: channel_id, text: "<@#{tech_lead}> please make a copy of the <#{ENV['INCIDENT_TEMPLATE']}|incident template> and consider starting a video call.")
+    slack_client.chat_postMessage(channel: channel_id, text: "<@#{tech_lead}> please make a copy of the <#{ENV['INCIDENT_TEMPLATE']}|incident template>.")
+  end
+
+  def self.start_meet!(channel_id)
+    message = slack_client.chat_postMessage(channel: channel_id,
+                                            text: "Join the <#{ENV['MEETS_LINK']}|incident call.>")
```

Comment:
Would be great to have a proper API for meets — this will do in the meantime!

Reply:
👌